### DISCUSSION TITLE
NKE floats

### DISCUSSION THREAD

**Discussion by ClareBellingham** [nvs-vocabs/R28#6] [General]
A place to discuss NKE controller boards

**Comment by ClareBellingham** [comment]
> Hello Clare,
>
> I had a discussion with NKE a few weeks ago about the controler board on
> Provor/arvor floats, here are the information I have :
>
> For PROVOR CTS5 floats :
> CONTROLLER_BOARD_TYPE_PRIMARY = ‘APMT’
> CONTROLLER_BOARD_TYPE_SECONDARY = ‘OSEAN’ or ‘USEA’
>
> For all other NKE floats :
> CONTROLLER_BOARD_TYPE_PRIMARY = ‘I535’
>
> About the Battery type :
>
> For the Arvor Deep floats :
> BATTERY_TYPE = ‘SAFT Lithium 14.5 V’
>
> For all others NKE floats :
>
> BATTERY_TYPE = ‘SAFT Lithium 11 V’
>
> I don't have the information for the other kind of floats, I would be
> happy to get it (we just put CONTROLLER_BOARD_TYPE_PRIMARY = ‘APF9’ or
> ‘APF11’ for the Apex floats)
>
> Should I update the github with this information? or should we create
> some sub-directories per float family ? :
> https://github.com/nvs-vocabs?q=argo&type=&language=&sort=
>
> Thanks in advance,
>
> Regards,
>
> Vincent
>
> --
> ---------------------------------------
>    Vincent BERNARD
>    Coriolis Data Center

**Comment by ClareBellingham** [comment]
update from ifremer
Hello Clare,

sorry for the delay,

I've just had a talk with Jean Philippe and here are the controller
boards we have for NKE floats :

METOCEAN, CTF1 , CTF2, CTF3, CTS2, CTS3, CTS4, CTS5, I535, OSEAN and USEA

Regards,

Vincent

**Comment by ClareBellingham** [comment]
primary and  secondary controller boards mentioned in https://www.frontiersin.org/articles/10.3389/fmars.2020.577446/full as NKE's new controller board (developed in partnership with LOV) equipped on Provor CTS5 e.g.

**Comment by delphinedobler** [comment]
Final investigation led to the list as in https://github.com/OneArgo/ArgoVocabs/issues/169#issuecomment-3790245508

This discussion can be closed.

**Comment by ClareBellingham** [comment]
closing discussion
